plugins/gossip/webapi.go

```diff
@@ -0,0 +1,90 @@
+package gossip
+
+import (
+	"net/http"
+
+	"github.com/iotaledger/hive.go/autopeering/peer"
+	"github.com/iotaledger/hive.go/crypto/ed25519"
+	"github.com/iotaledger/hive.go/identity"
+	"github.com/labstack/echo"
+	"golang.org/x/xerrors"
+
+	"github.com/iotaledger/goshimmer/packages/gossip"
+	"github.com/iotaledger/goshimmer/plugins/webapi"
+	"github.com/iotaledger/goshimmer/plugins/webapi/jsonmodels"
+)
+
+func configureWebAPI() {
+	webapi.Server().POST("gossip/neighbors", addNeighborsHandler)
+	webapi.Server().DELETE("gossip/neighbors", dropNeighborsHandler)
```

Comment:
should we add a GET route to get the neighbors from gossip? we can fetch the autopeering neighbors from `autopeering/neighbors`, but I guess those will not include manual peers

Reply:
Yeah, I think it's good to add such API